app/services/katello/pulp3/api/docker.rb

```diff
@@ -37,7 +37,9 @@ def self.tag_image_class
         end
 
         def api_client
-          api_client_class(PulpContainerClient::ApiClient.new(smart_proxy.pulp3_configuration(PulpContainerClient::Configuration)))
+          config = smart_proxy.pulp3_configuration(PulpContainerClient::Configuration)
```

Comment:
To fix this, Pulp will need to release all-new z-streams. I think we can live with this workaround for now. I'll make an issue when this PR merges.